src/main/java/org/openstreetmap/atlas/geography/atlas/change/serializer/FeatureChangeGeoJsonSerializer.java

```diff
@@ -207,15 +207,14 @@ else if (source instanceof Relation)
         return result;
     };
 
-    private static final String NULL = "null";
     private final Gson jsonSerializer;
 
     private static <T> void add(final JsonObject result, final String name, final T property,
             final Function<T, JsonElement> writer)
     {
         if (property == null)
         {
-            result.addProperty(name, NULL);
+            result.addProperty(name, (String) null);
```

Comment:
@yazad3 See here. It seems like explicitly adding a null property is not actually reflected in the final object. Gson just skips the property altogether when converting to string.